media-gfx/librecad/files/librecad-2.2.0_rc1-qt-5.15.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/librecad/src/lib/engine/lc_splinepoints.cpp b/librecad/src/lib/engine/lc_splinepoints.cpp
```

Comment:
1. These files are already in files/, no need to copy them if they don't need changes for the new version. Simply replace `${P}` in patches with the correct name.

2. Please don't remove the header with description, otherwise after a while it will be impossible to say what is the purpose of this patch.

Reply:
The old patch doesn't apply because the line numbers have changed, I think. That is why I created a new patch file, even though the contents are the same. Is there a better way?

I will add the header with description.

Reply:
If just the line numbers changed, the patch would still apply. But if context changed, like the includes nearby... I see how it wouldn't apply anymore.

Reply:
Ah, OK then the context must've changed. Because originally I did try to reuse the original patch, and only when it failed to apply I created a new one.

Reply:
I think you are right for one of the patches, will check and fix.

Reply:
You are absolutely right! I've removed one patch. The other remains relevant, though.